src/proxy-ui-api/frontend/src/views/Diagnostics/Diagnostics.vue

```diff
@@ -1,16 +1,216 @@
 <template>
   <v-layout align-center justify-center column fill-height elevation-0>
-    <div>Placeholder</div>
+    <v-card class="xrd-card diagnostic-card">
+      <v-card-title>
+        <span class="headline">{{$t('diagnostics.globalCongiguration.title')}}</span>
+      </v-card-title>
+      <v-card-text class="pt-4">
+        <table class="xrd-table">
+          <thead>
+            <tr>
+              <th class="status-column">{{$t('diagnostics.status')}}</th>
+              <th>{{$t('diagnostics.message')}}</th>
+              <th class="time-column">{{$t('diagnostics.previousUpdate')}}</th>
+              <th class="time-column">{{$t('diagnostics.nextUpdate')}}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-if="globalconf">
```

Comment:
This applies to timestamping and ocsp responders -cards as well. Should there be some sort if no globalconf is empty for some reason?

Reply:
It might be good to have some "no data found" text or something like that, but there should be a one consistent way that the whole app uses. Currently a "empty table" is the way it's done in all other places also. This probably should be improved at some point.
I added a loading indicator on the cards so at least the user sees if it's empty or loading.

Reply:
Yeah, it should be improved. I've added such infotext to a couple of table views. I guess we should make a component of that.